src/pke/lib/scheme/ckksrns/ckksrns-parametergeneration.cpp

```diff
@@ -316,16 +316,21 @@ void ParameterGenerationCKKSRNS::CompositePrimeModuliGen(std::vector<NativeInteg
                         }
                     }
                     catch (const OpenFHEException& ex) {
-                        OPENFHE_THROW(
-                            "COMPOSITE SCALING next prime sampling error. "
-                            "Try increasing scaling factor (scalingModSize) "
-                            "or decreasing first modulus size (firstModSize) "
-                            "or try another combination of firstModSize and scalingModSize.");
+                        std::string errMsg = "COMPOSITE SCALING next prime sampling error. ";
```

Comment:
This was my comment added to the other PR's review:
Are you sure that you want to override OpenFHEException caught in 4 places in this file? Also, the new error message is very long. I would throw a shorter error message combined with the original error message. And the new error message can be left in the same place commented out (for debugging purposes)

Reply:
I figured that if the other PR is merged first, this comment is resolved. I resolved it there. Otherwise, I can cherry-pick the commit and apply it to this PR. The intent was to separate these changes for semantics' sake.

Reply:
i am not sure what to look at as there are 3 PRs submitted, i don't know what will be merged first and what code will be overridden

Reply:
So, this one should be fixed first.

Reply:
@dsuponitskiy Done. Please see latest commits.